src/tray.rs

```diff
@@ -1,11 +1,53 @@
+use std::io::Cursor;
+
+use image::Rgba;
 use ksni;
 
 #[derive(Debug)]
-struct Tray;
+struct Tray {
+    icon: ksni::Icon,
+}
+
+impl Tray {
+    pub fn new() -> Self {
+        let rgba_image = {
+            let cursor = {
+                let image_bytes = include_bytes!("../assets/flakeshot_logo_dpi_96.png");
+                Cursor::new(image_bytes.to_owned())
```

Comment:
```suggestion
                Cursor::new(image_bytes)
```

You don't have to own the bytes as it also takes a reference. At the end its just a wrapper anyway.